src/shared-components/EventListCard/Modal/updateLogic.spec.ts

```
@@ -19,6 +19,8 @@ import { cleanup } from '@testing-library/react';
 dayjs.extend(utc);
 const FIXED_NOW = dayjs.utc(new Date(Date.UTC(2025, 0, 1, 10)));
 
+
+// Mock react-i18next so useTranslation works without React context
```

ESLint

[error] Delete `⏎`

(prettier/prettier)